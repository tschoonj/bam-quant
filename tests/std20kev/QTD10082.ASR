File: C:\xrfdaten\2014KW30\chris\std20kev\std10082.spe
Start: 21.07.2014 18:36:19
Stop: 21.07.2014 18:38:00
BESSY: 297.975362 297.941602
IONI: 3.087557E-11 3.108837E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 80691
Run: 100000
Dead: 19
ICR: 60610
OCR: 0
Accumulated BESSY: 59693.0814136
Accumulated Ioni: 6.21490765E-09
N Accumulation: 200
Normfactor:  3.9881384e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pb
exp_x 131.260406494141 131.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  82   2     10.542   2142911.      1385.     198.29
