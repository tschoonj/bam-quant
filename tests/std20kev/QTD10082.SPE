$DATA:
0 2000
18774
12149
5316
1659
379
56
11
4
3
2
3
0
1
1
0
7
164
1123
1379
1317
1244
1288
1255
1100
1123
1091
1013
1050
1104
998
993
942
1001
927
906
894
834
751
769
706
715
667
635
654
644
629
622
619
584
586
538
541
571
580
535
525
579
513
541
530
498
496
532
504
483
523
445
510
504
466
450
495
458
451
448
471
464
469
422
472
411
414
462
439
398
397
447
426
428
465
537
504
534
520
491
502
436
453
440
436
453
465
514
462
506
486
524
532
546
622
693
824
1195
1986
3015
4148
5194
5862
6133
6376
6066
5780
5037
3995
3001
2016
1354
905
701
627
622
621
593
610
551
510
443
466
445
411
409
398
401
415
418
444
427
453
442
410
405
435
465
476
474
474
441
414
427
411
372
337
372
329
333
352
312
361
384
347
388
354
338
368
372
329
324
360
364
349
354
362
355
391
340
360
317
330
313
330
328
310
298
303
319
309
289
322
300
291
290
258
314
283
308
280
316
304
291
292
281
286
319
283
298
303
279
273
293
318
306
285
298
274
283
289
284
307
273
302
291
314
272
288
303
311
301
283
245
293
274
283
306
313
315
267
299
276
278
282
272
293
293
268
324
297
262
260
286
299
291
295
287
292
265
309
331
287
313
309
285
285
266
279
283
280
302
272
310
277
286
280
297
301
307
286
263
297
296
323
277
288
263
299
310
313
305
314
306
303
302
333
297
308
296
275
296
306
301
302
311
320
333
317
337
348
385
361
393
348
331
351
312
331
274
277
330
311
302
287
308
289
288
301
322
287
304
305
317
299
337
312
281
313
310
329
299
304
318
323
353
307
338
276
301
311
339
306
294
287
301
309
292
334
312
338
310
333
326
361
338
316
327
308
326
334
336
332
324
362
338
322
333
342
339
366
324
316
378
340
346
346
385
426
429
463
465
506
522
520
459
456
448
414
396
393
345
361
359
391
367
322
401
396
339
375
387
425
376
367
430
385
389
440
391
403
443
418
485
488
522
543
581
630
650
752
674
749
704
705
666
664
661
744
776
888
1155
1569
2219
3226
4506
6069
7602
8918
9629
10009
9465
8621
7202
5701
4319
3065
2252
1650
1237
964
835
780
806
795
781
768
713
688
650
642
646
685
662
689
705
691
714
687
721
768
734
776
832
865
899
918
1013
1028
1000
1092
1034
1092
1161
1141
1165
1232
1332
1334
1601
1772
1923
2387
3199
4456
6620
9871
15080
22741
33649
48610
68423
92520
120983
149780
176254
195487
204182
200429
183598
158455
127191
95992
67980
44708
28626
17603
10293
5973
3699
2552
1924
1585
1366
1225
1135
980
956
894
845
805
765
730
714
773
757
823
847
1045
1279
1590
1969
2521
2987
3686
4102
4392
4455
4318
3856
3439
2940
2254
1742
1428
1148
984
859
790
719
714
730
672
767
719
751
769
747
758
755
860
826
869
924
861
985
1107
1275
1439
1751
2068
2532
3243
3726
4296
5010
5888
6798
7791
8740
9776
11235
12189
12980
13407
13227
12977
12853
12813
14702
18433
25666
37539
53691
74907
99571
126930
151652
170972
181877
183651
174326
156772
133458
108802
85044
64852
48186
36229
27586
21159
16791
13260
10886
9253
7952
7368
7315
6794
6713
6298
5776
5084
4385
3583
2883
2183
1695
1334
1037
876
771
753
721
733
755
688
664
612
602
538
433
385
333
265
240
186
167
165
182
139
177
145
165
157
150
176
146
150
150
147
179
190
139
157
178
172
187
189
188
187
206
205
220
230
287
352
434
459
570
673
746
797
768
844
804
794
657
580
525
558
512
538
659
897
1186
1790
2608
3711
5452
7181
9504
11728
14036
15754
17105
17390
16894
15447
13575
10977
8732
6630
4900
3829
2867
2447
2291
2194
2618
3090
3622
4057
4461
4855
5174
5038
4949
4637
4243
3638
2992
2421
1912
1426
977
676
537
341
276
237
187
149
137
133
155
146
170
220
260
376
404
494
506
598
614
650
595
543
473
387
341
240
185
154
101
71
49
33
22
23
13
14
20
10
9
10
10
10
9
15
9
12
11
7
7
8
11
12
10
13
11
7
10
5
7
9
6
12
12
4
4
7
8
11
14
5
11
6
11
4
7
8
7
7
12
13
9
10
4
10
8
15
9
5
20
9
9
8
6
5
7
10
7
7
8
7
10
3
8
3
8
9
10
8
7
10
11
5
10
5
12
7
8
11
6
8
9
4
7
10
5
3
5
8
8
3
8
5
5
4
7
5
4
8
8
7
9
12
6
4
10
11
5
4
11
9
9
5
14
10
8
9
12
10
7
6
7
10
5
8
7
12
9
7
8
8
11
11
10
9
11
17
11
11
10
14
12
17
14
11
17
20
18
15
20
14
19
19
17
20
26
30
24
28
21
30
20
29
35
41
42
36
50
46
46
66
69
62
88
71
95
77
86
96
140
128
141
117
144
163
174
180
162
180
184
159
164
176
137
155
149
144
115
112
78
72
80
61
47
37
30
30
26
15
23
8
13
16
10
7
11
8
9
8
8
7
1
10
10
10
16
16
25
43
37
51
84
95
113
116
160
204
243
257
306
323
320
326
311
339
312
272
243
183
149
132
88
78
62
40
44
24
28
20
19
15
17
5
9
13
8
10
14
12
15
15
20
38
25
35
36
39
40
28
40
46
51
40
26
51
44
23
13
24
23
14
11
11
13
7
12
8
8
7
5
7
4
8
3
11
9
6
9
8
9
14
6
16
20
12
14
15
25
21
27
49
41
48
41
50
51
57
59
61
96
105
90
117
147
199
191
247
303
390
413
533
514
584
590
628
631
608
591
485
454
447
410
327
254
201
165
138
104
89
71
65
62
42
40
39
34
23
23
23
25
28
15
19
17
14
13
13
10
13
27
15
20
17
19
19
24
18
16
26
14
10
10
5
11
12
5
8
11
10
4
3
4
3
5
13
9
10
15
7
10
10
5
5
8
8
13
20
18
21
29
17
19
29
33
23
38
39
47
61
50
51
45
65
83
78
86
107
105
130
139
187
196
250
331
285
306
332
345
367
333
362
332
275
233
218
219
189
137
114
99
89
77
63
57
58
60
55
43
45
53
29
32
26
22
26
19
18
21
18
13
11
8
11
7
11
6
12
5
4
6
3
7
5
6
2
7
3
6
1
5
1
3
1
3
2
2
2
3
2
3
2
4
0
2
1
2
3
2
3
3
2
2
2
5
2
5
6
12
1
4
5
8
3
7
7
14
10
11
12
8
13
9
19
22
21
24
36
38
47
38
54
41
56
53
59
49
58
39
59
35
27
44
25
29
28
19
16
25
16
18
31
17
18
15
19
21
20
19
18
16
13
9
7
4
8
5
2
2
6
3
3
3
3
6
2
3
1
2
1
1
1
5
3
3
5
1
1
3
2
4
2
1
0
0
1
0
1
1
0
0
1
2
0
0
0
1
2
1
0
2
1
1
0
0
0
1
0
1
1
1
0
1
1
1
1
0
2
0
0
1
0
1
3
1
2
2
3
8
3
3
4
4
0
6
2
3
1
0
1
1
2
0
0
0
4
2
1
3
4
2
3
1
4
4
1
4
2
1
1
2
0
2
0
0
1
0
2
0
0
2
1
0
1
2
3
0
3
0
0
0
1
0
4
3
1
0
0
0
1
2
0
1
0
2
0
1
0
0
1
1
0
0
0
0
0
0
1
1
0
0
0
1
0
0
0
0
0
0
0
0
0
1
1
0
1
0
1
1
0
0
1
0
0
0
0
0
1
1
0
0
0
2
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
1
2
0
1
1
1
0
0
0
1
0
2
0
0
1
1
0
0
0
0
0
0
1
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
1
1
0
0
0
0
1
1
0
1
0
0
2
1
1
0
1
1
3
0
0
2
1
1
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
2
0
0
0
0
1
0
0
0
1
0
0
0
1
0
0
0
0
1
1
0
1
3
2
1
0
3
1
2
2
3
1
3
0
1
0
0
0
3
0
0
0
0
0
0
0
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
3
0
0
1
0
0
0
1
1
0
0
0
1
1
1
0
0
0
0
1
0
0
0
1
0
1
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10082.spe
Start: 21.07.2014 18:36:19
Stop: 21.07.2014 18:38:00
BESSY: 297.975362 297.941602
IONI: 3.087557E-11 3.108837E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 80691
Run: 100000
Dead: 19
ICR: 60610
OCR: 0
Accumulated BESSY: 59693.0814136
Accumulated Ioni: 6.21490765E-09
N Accumulation: 200
Normfactor:  3.9881384e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pb
 
exp_x 131.260406494141 131.2604
exp_y 65.3237991333008 65.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  82   2     10.542   2142911.      1385.     198.29
