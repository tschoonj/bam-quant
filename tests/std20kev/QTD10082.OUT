     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:59  บ   
     บ  Spectrum: QTD10082.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        3 iterations done    บ   
     บ  ChiSquare =   180.6   last change =   .01%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -30.7 ๑   100.0   บ     -30.7 ๑      .3   บ   
     บ  GAIN (eV/ch)  บ     20.218 ๑   2.000   บ    20.218 ๑    .001   บ   
     บ det NOISE (eV) บ      107.1 ๑    40.0   บ     107.1 ๑     1.6   บ   
     บ  FANO factor   บ       .110 ๑    .050   บ      .110 ๑    .001   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Pb-L1                            293088. ๑    778.           บ   
     บ    L1M3       12.793    .37802      110793. ๑    295.    998.2  บ   
     บ              634.265    204.05         268.           9.85E-01  บ   
     บ    L1M2       12.306    .34544      101246. ๑    269.    501.9  บ   
     บ              610.177    201.22         283.           9.83E-01  บ   
     บ    L1N3       15.218    .10996       32229. ๑     87.    817.4  บ   
     บ              754.205    217.61         222.           9.87E-01  บ   
     บ    L1N2       15.101    .09081       26615. ๑     72.    888.5  บ   
     บ              748.419    216.97         226.           9.87E-01  บ   
     บ    L1O3       15.765    .03930       11519. ๑     34.    128.0  บ   
     บ              781.260    220.55         207.           9.88E-01  บ   
     บ    L1M5       13.377    .01893        5549. ๑     22.     24.6  บ   
     บ              663.149    207.40         250.           9.87E-01  บ   
     บ    L1M4       13.275    .01262        3698. ๑     19.     48.8  บ   
     บ              658.104    206.81         253.           9.86E-01  บ   
     บ    L1N5       15.453    .00301         882. ๑     15.    323.2  บ   
     บ              765.829    218.88         216.           9.88E-01  บ   
     บ    L1N4       15.427    .00190         558. ๑     15.    388.0  บ   
     บ              764.543    218.74         217.           9.88E-01  บ   
     บ                                                                 บ   
     บ  2 Pb-L2                           1651302. ๑   1527.           บ   
     บ    L2M4       12.614    .78003     1288072. ๑   1191.   1188.3  บ   
     บ              625.411    203.01         273.           9.85E-01  บ   
     บ    L2N4       14.764    .16802      277454. ๑    257.   1991.5  บ   
     บ              731.750    215.13         236.           9.87E-01  บ   
     บ    L2M1       11.349    .02148       35465. ๑     37.    301.8  บ   
     บ              562.844    195.53         314.           9.84E-01  บ   
     บ    L2N1       14.308    .00561        9266. ๑     17.     26.6  บ   
     บ              709.197    212.62         224.           9.87E-01  บ   
     บ    L2O4       15.178    .02276       37576. ๑     38.    819.2  บ   
     บ              752.227    217.39         224.           9.87E-01  บ   
     บ    L2O1       15.053    .00120        1986. ๑     15.    994.9  บ   
     บ              746.044    216.71         227.           9.87E-01  บ   
     บ    L2M3       12.127    .00090        1483. ๑     17.    604.4  บ   
     บ              601.324    200.17         288.           9.83E-01  บ   
     บ                                                                 บ   
     บ  3 Pb-L3                           2778907. ๑   1795.           บ   
     บ    L3M5       10.552    .69240     1924119. ๑   1243.    225.8  บ   
     บ              523.424    190.67         342.           9.80E-01  บ   
     บ    L3M4       10.450    .07873      218792. ๑    142.    165.4  บ   
     บ              518.379    190.04         307.           9.80E-01  บ   
     บ    L3N5       12.623    .14017      389517. ๑    252.   1188.3  บ   
     บ              625.856    203.07         273.           9.85E-01  บ   
     บ    L3N4       12.601    .01566       43524. ๑     33.   1185.2  บ   
     บ              624.768    202.94         274.           9.84E-01  บ   
     บ    L3M1        9.185    .04029      111967. ๑     75.    173.2  บ   
     บ              455.812    182.02         350.           9.71E-01  บ   
     บ    L3N1       12.143    .01012       28133. ๑     25.    601.7  บ   
     บ              602.115    200.26         288.           9.83E-01  บ   
     บ    L3O1       12.888    .00211        5864. ๑     17.    418.4  บ   
     บ              638.963    204.60         265.           9.85E-01  บ   
     บ    L3O5       13.015    .02051       56990. ๑     40.    187.9  บ   
     บ              645.245    205.33         261.           9.86E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                           24864. ๑    160.           บ   
     บ    L3O5       23.168    .29700        7524. ๑     49.      7.7  บ   
     บ             1147.399    257.08          71.           8.67E-01  บ   
     บ               21.104    .16596        4204. ๑     29.      4.5  บ   
     บ             1045.328    247.44         104.           9.27E-01  บ   
     บ    L2N1       25.231    .13288        3366. ๑     23.      8.0  บ   
     บ             1249.471    266.38          50.           7.96E-01  บ   
     บ    L3M5       25.316    .04788        1213. ๑     11.      5.4  บ   
     บ             1253.654    266.75          49.           7.93E-01  บ   
     บ    L3N5       27.380    .04285        1085. ๑      9.      6.1  บ   
     บ             1355.725    275.72          29.           7.19E-01  บ   
     บ    L3M1       21.002    .03774         956. ๑     12.      4.6  บ   
     บ             1040.283    246.95         105.           9.29E-01  บ   
     บ    L3N1       23.066    .03377         856. ๑     10.     10.8  บ   
     บ             1142.354    256.61          72.           8.70E-01  บ   
     บ               23.350    .02009         509. ๑      9.      1.4  บ   
     บ             1156.405    257.92          68.           8.61E-01  บ   
     บ               19.737    .01931         489. ๑     12.     47.7  บ   
     บ              977.716    240.84         130.           9.56E-01  บ   
     บ               25.413    .01798         455. ๑      8.      1.5  บ   
     บ             1258.476    267.18          48.           7.90E-01  บ   
     บ               22.858    .01744         442. ๑      9.     12.5  บ   
     บ             1132.081    255.66          75.           8.77E-01  บ   
     บ               21.801    .01728         438. ๑     10.      2.2  บ   
     บ             1079.787    250.74          92.           9.08E-01  บ   
     บ               25.720    .01696         430. ๑      7.      4.4  บ   
     บ             1273.624    268.53          45.           7.79E-01  บ   
     บ               24.922    .01560         395. ๑      8.     10.4  บ   
     บ             1234.152    265.00          54.           8.07E-01  บ   
     บ               27.783    .01518         384. ๑      6.      5.1  บ   
     บ             1375.695    277.44          26.           7.05E-01  บ   
     บ               23.567    .00983         249. ๑      8.      1.2  บ   
     บ             1167.148    258.91          65.           8.54E-01  บ   
     บ               25.631    .00880         223. ๑      7.      4.4  บ   
     บ             1269.220    268.14          45.           7.82E-01  บ   
     บ               21.901    .00612         155. ๑     10.      2.3  บ   
     บ             1084.748    251.21          90.           9.06E-01  บ   
     บ               23.965    .00548         139. ๑      8.      1.4  บ   
     บ             1186.819    260.71          61.           8.41E-01  บ   
     บ               25.214    .00544         138. ๑      7.      8.5  บ   
     บ             1248.609    266.30          50.           7.97E-01  บ   
     บ               22.694    .00511         129. ๑      9.      6.5  บ   
     บ             1123.979    254.90          78.           8.82E-01  บ   
     บ               24.758    .00457         116. ๑      7.      6.4  บ   
     บ             1226.051    264.27          55.           8.13E-01  บ   
     บ               29.528    .00345          88. ๑      4.      5.4  บ   
     บ             1461.980    284.75          14.           6.44E-01  บ   
     บ               27.562    .00290          73. ๑      5.      8.1  บ   
     บ             1364.731    276.49          27.           7.12E-01  บ   
     บ               23.949    .00279          71. ๑      8.      1.4  บ   
     บ             1186.042    260.64          61.           8.41E-01  บ   
     บ               27.070    .00252          64. ๑      6.      1.9  บ   
     บ             1340.407    274.39          31.           7.30E-01  บ   
     บ               29.932    .00245          62. ๑      4.      4.5  บ   
     บ             1481.950    286.42          13.           6.31E-01  บ   
     บ               23.248    .00228          58. ๑      8.      4.6  บ   
     บ             1151.360    257.45          70.           8.65E-01  บ   
     บ               19.635    .00220          56. ๑     11.     23.5  บ   
     บ              972.671    240.34         132.           9.58E-01  บ   
     บ               20.900    .00215          54. ๑     10.      3.5  บ   
     บ             1035.238    246.47         107.           9.32E-01  บ   
     บ               26.317    .00198          50. ๑      6.      3.4  บ   
     บ             1303.164    271.14          38.           7.57E-01  บ   
     บ               22.756    .00198          50. ๑      9.      8.6  บ   
     บ             1127.036    255.19          77.           8.80E-01  บ   
     บ               25.618    .00193          49. ๑      7.      4.5  บ   
     บ             1268.579    268.08          46.           7.82E-01  บ   
     บ               28.381    .00178          45. ๑      5.      2.1  บ   
     บ             1405.235    279.96          21.           6.84E-01  บ   
     บ               24.860    .00160          41. ๑      7.      8.8  บ   
     บ             1231.100    264.73          54.           8.09E-01  บ   
     บ               26.924    .00143          36. ๑      6.      1.5  บ   
     บ             1333.172    273.76          33.           7.35E-01  บ   
     บ               27.779    .00142          36. ๑      5.      5.1  บ   
     บ             1375.474    277.42          26.           7.05E-01  บ   
     บ               21.983    .00117          30. ๑      9.      2.4  บ   
     บ             1088.793    251.59          89.           9.03E-01  บ   
     บ               23.465    .00112          28. ๑      8.      1.3  บ   
     บ             1162.103    258.44          67.           8.57E-01  บ   
     บ               25.104    .00106          27. ๑      7.      9.6  บ   
     บ             1243.158    265.81          51.           8.01E-01  บ   
     บ               27.966    .00103          26. ๑      5.      4.6  บ   
     บ             1384.701    278.21          24.           6.98E-01  บ   
     บ               21.491    .00101          26. ๑     10.      2.3  บ   
     บ             1064.469    249.28          97.           9.17E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
